packs/utilities/templates/_group/volume.tpl

```diff
@@ -0,0 +1,11 @@
+[[/* iterate over `var.volumes` to create Volumes */]]
```

Comment:
What about the `volume_mount` block?

Reply:
Good one, added a `volume_mount` template helper